Integrating chat

rushil-arun opened the pull request.

rushil-arun committed:
Hooked up chats to backend, as well as change username feature

rushil-arun committed:
Hooked up chats to backend, as well as change username feature

rushil-arun committed:
Integrated overall chat screen. Going to implement database retrieval for Zustand in another branch.

rushil-arun committed:
Merge branch 'integratingChat' of https://github.com/SlysDev/friend-ly into integratingChat

rushil-arun committed:
Merge branch 'develop' into integratingChat

rushil-arun committed:
Finished connected profile/edit profile screens, along with chat messages. Need to fix chat timestamps before pull request

rushil-arun committed:
Finished integration of chat and profile. Will begin integration of posts next. TODO: Clean up all username retrieval endpoint.

rushil-arun committed:
Merge branch 'develop' into integratingChat

elise-p reviewed (commented):
Overall, I think the API stuff and logic looks good. But I just wonder if the token stored in requests.rest is encrypted, and what's being stored in the token.

jeremykamber committed:
feat: add creating chat and adding participants to convo functionality

- Implemented AddPeopleView and AddChatView components with search functionality and user selection.
- Integrated (mock) API calls to fetch friends and add selected users to chat.
- Enhanced ChatMessagesView to navigate to AddPeopleView with existing members.
- Updated ChatsView to include navigation to AddChatView.
- Added expo-blur dependency for iMessage-like UI enhancements.

jeremykamber committed:
Resolve merge conflict in integratingChat betw. add chat/participant frontend changes and backend modifications
